src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead tr');
+
+thead.addEventListener('click', event => {
+  const index = [...thead.children].indexOf(event.target);
+  const tbody = document.querySelectorAll('table tbody tr');
+
+  const sortedEmployees = [...tbody]
+    .sort((a, b) => {
+      const emloyeeA = a.children[index].innerText
+        .replace(/[^0-9a-z]/gi, '');
+      const emloyeeB = b.children[index].innerText
+        .replace(/[^0-9a-z]/gi, '');
+
+      return !isNaN(+emloyeeA)
```

Comment:
how about checking if typeof `string` instead of converting it to a number and then checking if it's not NaN?